src/types/onyx/Session.ts

```diff
@@ -2,7 +2,7 @@ import * as OnyxCommon from './OnyxCommon';
 
 type Session = {
     /** The user's email for the current session */
-    email?: string;
+    email: string;
```

Comment:
Add comment describing the property

Reply:
@blazejkustra It's already has the description